gap/Parser.gi

```diff
@@ -581,31 +581,28 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
         @EndAutoDocPlainText := function()
             plain_text_mode := false;
         end,
-        @URL := function( )
-            if not IsBound( tree!.TitlePage.TitleComment ) then
-                tree!.TitlePage.TitleComment := [ ];
-            fi;
-            tree!.content := tree!.TitlePage.TitleComment;
-
-            Add( tree, "<URL>" );
-            Add( tree, current_command[ 2 ] );
-            Add( tree, "</URL>" );
-        end
```

Comment:
Seems you decided to remove @URL? But then it should also be removed from `doc/Comments.xml:432`, no?